src/tile/FeaturedTile.js

```diff
@@ -0,0 +1,133 @@
+import React, { PropTypes } from 'react';
+import {
+	TouchableOpacity,
+	View,
+	Image,
+	StyleSheet,
+	Dimensions,
+} from 'react-native';
+import Text from '../text/Text';
+import Icon from '../icons/Icon';
+
+const FeaturedTile = ({
+	title,
+	icon,
+	caption,
+	imageSrc,
+	onPress,
+	activeOpacity,
+	containerStyle,
+	imageContainerStyle,
+	overlayContainerStyle,
+	titleStyle,
+	captionStyle,
+	width,
+	height,
+}) => {
+	if (!width) {
+		width = Dimensions.get('window').width;
+	}
+	if (!height) {
+		height = width * 0.8;
```

Comment:
1. The tile is not square by default. I can change it if need be
2. The height is available as a prop for customizing if anyone wants to change to specific height